lua/starfall/libs_cl/render.lua

```diff
@@ -1920,14 +1924,15 @@ end
 -- @param Angle angle Orientation of the box
 -- @param Vector mins Start position of the box, relative to origin.
 -- @param Vector maxs End position of the box, relative to origin.
-function render_library.draw3DWireframeBox(origin, angle, mins, maxs)
+-- @param boolean? writeZ Optional should the line be drawn with depth considered (default: false)
+function render_library.draw3DWireframeBox(origin, angle, mins, maxs, writeZ)
```

Comment:
Why's this one default false?

Reply:
Question is: Why are the other two `true` by default?  
It breaks backwards compat, doesn't adhere to how the respective GMod functions work and introduces unnecessary `if` statements.
Also, personally I don't really like true-by-default params.

Reply:
Ah nevermind, Starfall used `true` before, didn't notice that ;p

Reply:
I'm not sure why the writeZ values were not consistent to begin with. Having draw3DWireframeBox default to false maintains original behavior.

Reply:
I'm fine with breaking original behavior to keep these consistent. Lets make it default true.